epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -88,21 +87,30 @@ <h2>How EPUB conforming to EPUB accessibility guidelines cover European Accessib
 
 			<ul>
 				<li>discoverability of accessibility features of publications;</li>
-				<li>evaluation and certification of accessible EPUB publications.</li>
+				<li>conformance to Web Content Accessibility Guidelines;</li>
+				<li>specific EPUB requirements for accessibility;</li>
+				<li>evaluation and conformance reporting of accessible EPUB publications.</li>
 			</ul>
 
 			<p>Providing accessibility metadata facilitates informed decisions about the usability of an EPUB
 				publication. Consumers can examine the characteristics of the content and decide whether an EPUB
 				publication is appropriate for each need.</p>
 
-			<p>While it has always been possible to create EPUB publications with a high degree of accessibility, this
-				document also establishes formal requirements for contents to be certified as accessible. These
+			<p>While it has always been possible to create EPUB publications with a high degree of accessibility, 
+				this document establishes the formal accessibility requirements that content must meet to be declared 
```

Comment:
This paragraph also looks like a copy. It's talking about "this document" establishing the requirements for the EPUB Accessibility standard.

I think this whole section needs some rewriting to focus more on how the EPUB Accessibility standard matches the requirements of the EAA. Otherwise, maybe the title needs changing as I don't get a connection from this text to the EAA.